python/nano/test/neural-compressor/pytorch/test_trainer_quantize.py

```diff
@@ -142,3 +158,26 @@ def test_trainer_quantize_inc_ptq_customized(self):
             assert loaded_qmodel
             out = loaded_qmodel(x)
             assert out.shape == torch.Size([256, 10])
+
+    def test_trainer_quantize_inc_ptq_customized_no_forward(self):
+        # Test if a Lightning Module not compiled by nano works
+        train_loader_iter = iter(self.train_loader)
+        x = next(train_loader_iter)[0]
+        trainer = Trainer(max_epochs=1)
+
+        qmodel = trainer.quantize(self.user_defined_pl_model_no_forward,
+                                  calib_dataloader=self.train_loader,
+                                  inference_method_name="predict")
+        assert qmodel
+        out = qmodel(x)
+        assert out.shape == torch.Size([256, 10])
+
+        # save and load
+        with tempfile.TemporaryDirectory() as tmp_dir_name:
+            trainer.save(qmodel, tmp_dir_name)
+            loaded_qmodel = trainer.load(tmp_dir_name,
+                                         self.user_defined_pl_model_no_forward,
+                                         inference_method_name="predict")
```

Comment:
Can we save `inference_method_name` in the meta file?

Reply:
yes, it's critical for the usage of `model_int8.inference_method_name`

Reply:
then there is no need to have the user the specify `inference_method_name` when loading the model, yes?